windows/code/RNDeviceInfoCPP.h

```diff
@@ -58,6 +119,23 @@ namespace winrt::RNDeviceInfoCPP
         return false;
       }
     }
+	
+    REACT_SYNC_METHOD(getDeviceTypeSync);
+    std::string getDeviceTypeSync() noexcept
```

Comment:
Since you're in here, how about adding a device type for console?

Reply:
if (winrt::Windows::System::Profile::AnalyticsInfo::VersionInfo().DeviceFamily() == L"Windows.Xbox")
{
   return "TV";
}

Reply:
Since I'll need to add this to
`export type DeviceType = 'Handset' | 'Tablet' | 'Tv' | 'Desktop' | 'unknown';`

Should this be a bit more generic like `Console` or `GameConsole`.

Since using "Xbox" here doesn't really feel like it aligns with the other device types?

Reply:
Talked offline, updated to GameConsole.

Reply:
I like that. It's not really TV it does imply a specific controller type, and these types are the high-level sort-of-bundle of expected features so seems right